frontend/src/views/Users.vue

```diff
@@ -1,5 +1,7 @@
 <template>
   <div >
+    <keep-alive include="UserSearch">
       <router-view></router-view>
+    </keep-alive>
```

Comment:
This keeps the "UserSearch" component "alive" when routing between Views. This means that you can open a User's Details, make an edit, and then click the browser's "Back" button to return to your search results: it even saves the form data.

Normally I try to keep my Pull Requests as simple as possible, but this seems like related functionality and it's a one-line change for an immediate improvement.

Reply:
Is it the behavior we want, to save the data when using the browser Back button? 
Does it actually send the data to Keycloak? or does it just save it in the browser?

Reply:
It's not persisted anywhere (since it's just search inputs) just stored in the browser cache
See: https://vuejs.org/v2/guide/components-dynamic-async.html

Reply:
This means that you can open a User's Details, make an edit, and then click the browser's "Back" button to return to your search results: it even saves the form data. => so that means the Search form, not the User Details form? I get it now